inc/ocf_err.h

```diff
@@ -44,6 +44,9 @@ typedef enum {
 	/** Cache metadata found on device */
 	OCF_ERR_METADATA_FOUND,
 
+	/** Metadata on the device doesn't match with metadata in DRAM */
+	OCF_ERR_SUPERBLOCK_MISMATCH,
```

Comment:
You also need to update `OcfErrorCode` in pyocf.

Reply:
updated